src/js/helpers/graph/declarationHelper.mock.js

```diff
@@ -146,6 +146,116 @@ export const compositeNodeWithTransitions = {
   },
 };
 
+
+export const compositeWithMultipleLevelEndNodes = {
```

Comment:
It looks that `compositeWithMultipleLevelEndNodes` and `compositeWithNodeWithAllTransitions` are exactly the same.

Reply:
They aren't, they have different subtasks. However while checking this I noticed that the code mutates the input graph, so I fixed it.